Controllers/ServiceController.cs

```diff
@@ -1,16 +1,205 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Configuration;
+using System.Data.SqlClient;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Web.Http;
+using ulacit_bnb.Models;
 
 namespace ulacit_bnb.Controllers
 {
     [Authorize]
     [RoutePrefix("api/service")]
     public class ServiceController : ApiController
     {
+        //GET GetId
+        [HttpGet]
+        public IHttpActionResult GetId(int id)
+        {
+            Service service = new Service();
+            try
+            {
+                using (SqlConnection sqlConnection = new SqlConnection(ConfigurationManager.ConnectionStrings["ULACITBnB"].ConnectionString))
```

Comment:
This line could be separated into the following:
```c#
SqlConnection sqlConnection = new SqlConnection(ConfigurationManager.ConnectionStrings["ULACITBnB"].ConnectionString)
using (sqlConnection)
```
Moving the declaration of the variable outside the `using` the code looks more readable, are you agree?